license_sh/helpers.py

```diff
@@ -315,6 +318,29 @@ def get_dependency_tree_with_licenses(
     return dependency_tree, unknown_licenses, has_issues
 
 
+def override_packages(
+    dep_tree: PackageNode,
+    overriden_packages: LanguageOverridenPackages,
+    analyze: bool = False,
+) -> PackageNode:
+    for node in PreOrderIter(dep_tree):
+        new_license = None
+        new_license_text = None
+        node_name, node_version = getattr(node, 'name', None), getattr(node, 'version', None)
+        node_name_with_version = f"{node_name}=={node_version}"
```

Comment:
This will work for Python only. What about `npm` and other package managers? + Please add tests for those we well.

Reply:
This will work for all. This is just pipenv way of expressing it. The same way ignored packages uses just this